ubuntu_install_requirements.sh

```diff
@@ -21,10 +21,15 @@ sudo apt -y install \
   python3-pip \
   python3-setuptools \
   zlib1g-dev \
-  libssl1.0-dev \
   openssh-server \
   wget
 
+if [[ ${OS_VERSION} -eq 18 ]]; then
```

Comment:
See the above comment, 
The comparison can also be made against 18.04, avoiding having a second variable.

Reply:
see my comment above